content/en/docs/concepts/sampling/index.md

```diff
@@ -53,8 +53,8 @@ inspecting the trace as a whole.
 
 For example, the most common form of head sampling is
 [Consistent Probability Sampling](/docs/specs/otel/trace/tracestate-probability-sampling/#consistent-probability-sampling).
-It may also be referred to as Deterministic Sampling. In this case, a sampling
-decision is made based on the trace ID and a desired percentage of traces to
+This may also be referred to as Deterministic Sampling. In this case, a sampling
```

Comment:
If you're up for it, we are trying to root out the use of "may" in the docs, so you could change any occurrences on this page to "might" and earn our sincere gratitude!

Reply:
it is referred to as Deterministic Sampling, so I changed this one to "is" but will update the others to use "might"